js/packages/wallet-adapter-mobile/src/adapter.ts

```diff
@@ -44,6 +52,10 @@ function getPublicKeyFromAddress(address: Base64EncodedAddress): PublicKey {
 }
 
 export class SolanaMobileWalletAdapter extends BaseMessageSignerWalletAdapter {
+    readonly supportedTransactionVersions: Set<TransactionVersion> = new Set(
+        // FIXME(#244): We can't actually know what versions are supported until we know which wallet we're talking to.
+        ['legacy', 0],
+    );
```

Comment:
cc/ @jstarry, @sdlaver. This is the awkward bit; MWA plugin doesn't know what transaction versions it supports until it actually associates with a particular mobile wallet app. Despite that, I thought it impossible to do anything other than to tell the _type system_ that we support all versions.